openedx/features/course_experience/views/course_reviews.py

```diff
@@ -59,7 +62,8 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         context = {
             'course': course,
             'course_url': course_url,
-            'course_reviews_provider_fragment': course_reviews_provider_fragment
+            'course_reviews_provider_fragment': course_reviews_provider_fragment,
+            'is_enrolled': is_enrolled
```

Comment:
Nit: we add a comma even on the last line of dicts or lists, so that merges are cleaner.